tox.ini

```diff
@@ -1,11 +1,13 @@
 [tox]
-envlist = py{27,35}-django111,py{36,37}-django{111,20,21}
+envlist = py{27,35}-django111,py{35}-django{20,21,22}
+
 
 [testenv]
 deps =
-    django111: Django>=1.11,<2
+    django111: django==1.11
```

Comment:
Why did this need to be changed?  We normally want to test against the latest 1.11.x release.